lametro/templates/lametro/index.html

```diff
@@ -136,5 +136,40 @@ <h2>
             collapseActivity();
             return false;
         });
+
+        $('#search-bar').autocomplete({
+          serviceUrl: '{% url "autocomplete" %}',
+          paramName: 'query',
+          transformResult: function(response) {
+            var res = JSON.parse(response);
+            if (res.status_code == 500 || res.termHints.length < 1) {
+              return {
+                suggestions: [{'value': 'No topics found', 'data': 'na'}]
+              };
+            } else {
+              return {
+                suggestions: $.map(res.termHints, function(d) {
+                    return {'value': d.name, 'data': d.id};
+                  }
+                )
+              };
+            }
+          },
+          onSearchError: function(query, jqXHR, textStatus, errorThrown) {
+            return textStatus + ' : ' + errorThrown;
+          },
+          onSelect: function(suggestion) {
```

Comment:
if 'No topics found' is clicked, it populates the search bar but doesn't lead anywhere.

Reply:
i think i'd prefer it to do nothing if clicked. is that possible?

Reply:
i think i dug into this with no success last week, but i'll look again with fresh eyes